src/results/utils.rs

```diff
@@ -208,6 +208,21 @@ impl Serialize for FingerPrint {
     }
 }
 
+/// Split string into indent and rest texts.
+pub fn split_indent<'a>(line: &'a str) -> (&'a str, &'a str) {
+    match line.find(|c: char| !c.is_whitespace()) {
+        Some(p) => {
+            let pair = unsafe {
```

Comment:
This doesn't actually require unsafe code, since you can use the [`split_at`](https://doc.rust-lang.org/std/primitive.str.html#method.split_at) method of the [`str`](https://doc.rust-lang.org/std/primitive.str.html). Something like this would be great, and we would avoid unsafe code:
```rust
pub fn split_indent<'a>(line: &'a str) -> (&'a str, &'a str) {
    match line.find(char::is_alphanumeric) {
        Some(p) => line.split_at(p),
        None => ("", line),
    }
}
```